### DISCUSSION TITLE
Organization/Project name

### DISCUSSION THREAD

**Discussion by meichthys** [TriliumNext/Notes#3] [General]
I'm fine with TriliumNext if we want to stick with that, but I wanted to re-iterate my proposed alternative in case my initial proposal was buried in the [Trilium Maintenance Mode](https://github.com/zadam/trilium/issues/4620) comment thread:

In keeping with the spirit of the existing project, I would like to propose a new name of:

## Trilidium
I think it would be a good name for a few reasons:

- Trillidium is a species of Trillium (like the new project would be a species or descendent of Trilium)
- The one "L" spelling of Trillidium would match the one "L" spelling of Trillium which zadam chose for the current project. This would also give our new project a very unique namespace.
- Trilidium is similar enough to understand that it must be related to Trilium, yet different enough to indicate that it is something different

**Comment by eliandoran** [comment]
@meichthys , as I wrote in the original thread, I am also in favor of using this name.

The first thing that's important to me about this name change is that it will sound more cohesive. `Trilidium/Notes` sounds and feels better than `TriliumNext/Notes`.

The only downside that is see is that it does not make a direct connection to Trilium Notes as TriliumNext does.

Honestly, my hope will be that we would be able to "recover" the Trilium name if zadam would be willing, after we prove that the project is mature enough to carry over his work. For me, the name `Trilium` is a bit more melodic, not sure if it's a language bias or simply the fact that I got used to it.

**Reply by zadam** [reply]
I'm happy to yield the name "Trilium" once the new project properly succeeds the original (stable releases, roughly similar feature set, migration path from the original) and proves its sustainability. The original could be then renamed to Trilium Legacy. In that case, "Trilium Next" seems like a good transitory name.

The reason I'd like to keep the original name (and the repository) for now is that the future of Trilium Next is quite uncertain at this moment. It has momentum and big plans, but it might end up nowhere. It might also end up as its own product, very different from Trilium<sup>1</sup>. I sincerely hope it will succeed, though!

<sup>1.</sup> historical tidbit: Trilium started as a webapp companion to Notecase Pro

**Reply by meichthys** [reply]
Although I do really like the name, I'm not hard-set on `Trilidium`, so with Zadam's input, I'm fine with settling on `TriliumNext` with the idea that we could eventually assume the original name `Trilium` @eliandoran Do you want to mark this discussion as complete?

**Reply by eliandoran** [reply]
@meichthys , personally I'm perfectly happy with keeping Trilium Next for now and then integrate with Trilium Notes once we reach maturity. 🚀

**Comment by perfectra1n** [comment]
I agree with your statement @eliandoran, `Trilidium/Notes` almost sounds like an "upgrade" to `Trilium Notes`, and I also prefer it to `TriliumNext/Notes`, if my vote counts for anything lol.

**Comment by SiriusXT** [comment]
Perhaps due to language habits, I feel that Trilium next is simpler and easier to remember, and it conveys that this is a further version that may have new features, improvements or higher performance. Trilidium seems too long as a name, and spelling may be a problem for the unfamiliar.

**Comment by deajan** [comment]
I second @SiriusXT on this. Not being a native english speaker myself, `Trilidium` sounds like I will loose some letters while trying to remember / write that name.
I understand that trilidium is a sub-species of trilium plant, but even google will "automagically" redirect trilidium to trilium since there is not enough pages with that name, making the name less good for "being found" easily.

I'd go for TriliumNext / Trilium-ng or the alike.

PS: I'm old enough for trilidium name to make me think about Uridium on Commodore 64 ;) Good memories

**Comment by zerebos** [comment]
I personally don't feel strongly towards either Trilidium or Trilium Next. If I _had_ to choose, I would probably choose Trilidium. Like Elian though, there's the hope that at some point we may inherit the Trilium name.

**Comment by eliandoran** [comment]
Closing with the following conclusions:

- Will keep the Trilium Next name for now. It is perfectly indicating the ongoing nature of the development.
- If/when we manage to reach maturity we will discuss with zadam to use the Trilium Notes name with his blessing.

**Comment by eliandoran** [comment]
For additional context see: https://github.com/TriliumNext/Notes/issues/195
